net-im/mattermost-desktop-bin/mattermost-desktop-bin-4.6.2.ebuild

```diff
@@ -0,0 +1,75 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+MY_PN="${PN%-*}"
+
+inherit desktop
+
+DESCRIPTION="Mattermost Desktop application"
+HOMEPAGE="https://about.mattermost.com/"
+
+SRC_URI="
+	amd64? ( https://releases.mattermost.com/desktop/${PV}/mattermost-desktop-${PV}-linux-x64.tar.gz )
+	x86?   ( https://releases.mattermost.com/desktop/${PV}/mattermost-desktop-${PV}-linux-ia32.tar.gz )
+"
+
+LICENSE="Apache-2.0 GPL-2+ LGPL-2.1+ MIT"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+
+RDEPEND="
+	app-accessibility/at-spi2-atk:2
+	app-accessibility/at-spi2-core:2[X]
+	dev-libs/expat
+	dev-libs/nss
+	media-libs/alsa-lib
+	net-print/cups
+	sys-apps/dbus
+	x11-libs/gtk+:3[X]
+	x11-libs/libXScrnSaver
+"
+
+QA_PREBUILT="
+	opt/mattermost-desktop/mattermost-desktop
+	opt/mattermost-desktop/libnode.so
+	opt/mattermost-desktop/libffmpeg.so
+	opt/mattermost-desktop/libGLESv2.so
+	opt/mattermost-desktop/libEGL.so
+"
+
+DOCS=(
+	NOTICE.txt
+	README.md
+)
+
+S="${WORKDIR}"
+
+src_install() {
+	# adjust `S` based on architecture
+	if use amd64; then
+		S="${WORKDIR}/mattermost-desktop-${PV}-linux-x64"
+	elif use x86; then
+		S="${WORKDIR}/mattermost-desktop-${PV}-linux-ia32"
+	fi
+	cd "${S}" || die
```

Comment:
No you can't change `S` in a local phase, but what you can do and what I was after, is `cd` to proper directory based on the `if-else`.

Reply:
Done it, will force push the commit.